client/src/data/types/request.ts

```diff
@@ -11,4 +12,5 @@ export default interface Request {
     fulfilledAt?: Date;
     deleted?: boolean;
     fulfilled?: boolean;
+    matchedDonations?: DonationFormContributionTuple[];
```

Comment:
nit: I'd just leave it as DonationFormContribution, not sure if it should be called a tuple